src/feature/flow_chart_panel/hooks/useAddNewNode.tsx

```diff
@@ -42,13 +53,19 @@ export const useAddNewNode = () => {
     if (funcName === "CONSTANT") {
       nodeLabel = "2.0";
     } else {
-      const numOfThisNodesOnChart = nodes.filter(
-        (node) => node.data.func === funcName
-      ).length;
-      nodeLabel =
-        numOfThisNodesOnChart > 0
-          ? `${funcName}_${numOfThisNodesOnChart}`
-          : funcName;
+      // Commented out for now for performance reasons.
```

Comment:
Could keep a table of the node counts in `useFlowChartGraph`, updating when adding/deleting, but that's quite annoying... As it was before though it was causing the sidebar to rerender constantly while nodes were being dragged so I would prefer not to bring it back as is.

Reply:
This change was made specifically to ensure each node has a unique label. Instead of using the node count, we could consider using a `uuid` with two or three characters. What are your thoughts on this approach? Or maybe we can drop this if this causes no issue in ctrls panel or any other places.

Reply:
The uuid approach could be good, much easier to implement, although slightly worsens UX (also possibility for collisions?) 
As it is now, nodes having the same label definitely makes the ctrls panel harder to use, so we should probably figure something out